### PR TITLE
fix: remove prescision limits on Product.Item custom calculations

### PR DESCRIPTION
**Pull request by madcalf**
## What changed?
Remove the precision limits in the Product.Item custom field calculations. 

Follow up to #70

## How can you test this?
1. Run yarn develop to start the Strapi Admin
2. In the Content Manager, Add a new **Product.Item**
3. Add **Volume**, **Weight**, and **Value** components and fill in the required fields. (Using data from the Product Info spreadsheet will help by providing values that can be easily confirmed). Save the Item and verify that the following fields are now populated with the correctly calculated values and that the values (except currency) are not rounded:
  **Weight**
`itemWeightKg`
`countPerKg`
  **Volume**
`itemVolumeCBCM`
`countPerCBM`
  **Value**
`pricePerItemUSD`

4. Update an existing item with new values and confirm that the above calculations update appropriately.

### CODE REVIEW

**Review by corasaurus-hex — APPROVED** [review]
Seems good!

File: src/functions/content-types/product/item.ts [hunk 1]
```diff
@@ -94,6 +88,6 @@ function calculateValueFields(data) {
   // currently USD is the only unit available.
   const { packagePrice, countPerPackage } = data;
   let pricePerItemUSD = packagePrice / countPerPackage;
-  pricePerItemUSD = parseFloat(pricePerItemUSD.toFixed(2));
+  pricePerItemUSD = Math.round(pricePerItemUSD * 100) / 100;
```
Comment (jtfairbank) [inline src/functions/content-types/product/item.ts:91]: **question:** @madcalf I think I was wrong in the comments on the other PR.  `.toFixed(n)` will round so either way should be fine.  Go with the one that reads cleaner to you?

**Review by jtfairbank — APPROVED** [review]
**praise:** thanks for coming back to this one @madcalf!

One specific question on a code line, but merge in when you feel it's ready!

File: src/functions/content-types/product/item.ts (hunk 1 above)
Comment (jtfairbank) [reply src/functions/content-types/product/item.ts:91]: @madcalf sounds good!

**Pull request merged**